src/main/java/org/csanchez/jenkins/plugins/kubernetes/pipeline/ContainerStepExecution.java

```diff
@@ -24,10 +25,26 @@
 
     private static final transient Logger LOGGER = Logger.getLogger(ContainerStepExecution.class.getName());
 
-    private final ContainerStep step;
+    @Inject(optional=true)
```

Comment:
Again, no, you are not `Inject`ing anything. This is a little more confusing since you are extending `AbstractStepExecutionImpl`, which _can_ support Guice, but (as of #124) using the nondeprecated `super` constructor overload which does not.

Whether there is actually any reason to be extending `AbstractStepExecutionImpl` is another question. Normally you would just extend `StepExecution` directly. The only reason to continue to extend `AbstractStepExecutionImpl` is to maintain compatibility of `program.dat` deserialization across restarts when the plugin is updated—Java Serialization treats a change of superclass as unconditionally incompatible, even when `serialVersionUID` is maintained. But if I understand correctly the issue that this whole PR is addressing, it is that this step never worked across restarts to begin with! Thus there can be no _new_ problem introduced by getting rid of `AbstractStepExecutionImpl`.